WordPress/Classes/Networking/BlogServiceRemote.h

```diff
@@ -4,9 +4,14 @@
 
 typedef void (^OptionsHandler)(NSDictionary *options);
 typedef void (^PostFormatsHandler)(NSDictionary *postFormats);
+typedef void (^MultiAuthorCheckHandler)(BOOL isMultiAuthor);
 
 @protocol BlogServiceRemote <NSObject>
 
+- (void)checkMultiAuthorForBlog:(Blog *)blog
+                        success:(MultiAuthorCheckHandler)success
+                        failure:(void (^)(NSError *error))failure;
```

Comment:
Maybe, but if we want to adopt typedefs for blocks I think we should make that a separate issue and make the change through out.  Let's keep to the existing pattern for now.